frontend/src/app/uploads/DeviceLookup/DeviceLookup.tsx

```diff
@@ -0,0 +1,261 @@
+}
+
+interface Props {
+  formTitle: string;
+  swabOptions: Array<SwabType>;
+  deviceOptions?: DeviceType[];
+}
+
+const DeviceLookup = (props: Props) => {
+  const [device, updateDevice] = useState<Device | undefined>();
+
+  const [copiedSlug, setCopiedSlug] = useState<string>();
+
+  useEffect(() => {
+    const timeout = copiedSlug
+      ? setTimeout(() => {
+          setCopiedSlug(undefined);
+        }, 3000)
+      : undefined;
+    return () => {
+      if (timeout) {
+        clearTimeout(timeout);
+      }
+    };
+  }, [copiedSlug]);
+
+  async function copySlug(slug: string) {
+    try {
+      await navigator.clipboard.writeText(slug);
+      setCopiedSlug(slug);
+    } catch (e: any) {
+      console.error(e);
+    }
+  }
+
+  const onChange = () => {};
+
+  const getDeviceOptions = () =>
+    props.deviceOptions
+      ? props.deviceOptions
+          .map((deviceType) => ({
+            label: deviceType.name,
+            value: deviceType.internalId,
+          }))
+          .sort((a, b) => a.label.localeCompare(b.label))
+      : [];
+
+  const getDeviceFromDeviceType = (device?: DeviceType): Device | undefined => {
+    return device
+      ? {
+          internalId: device.internalId,
+          name: device.name,
+          manufacturer: device.manufacturer,
+          model: device.model,
+          swabTypes: device.swabTypes?.map((swab) => swab.internalId),
+          loincCode: device.loincCode,
+        }
+      : undefined;
+  };
+
+  const getCopyToClipboardButton = (
+    copiedAttribute: string | undefined,
+    label: string
+  ) => {
+    return (
+      <div className="copy-button-container copy-button-tooltip">
+        {device && (
+          <button
+            className="usa-button usa-button--unstyled"
+            onClick={() => copiedAttribute && copySlug(copiedAttribute)}
+            aria-label={`${label}`}
+          >
+            <FontAwesomeIcon
+              icon={copiedSlug === copiedAttribute ? faCheck : faCopy}
+            />
+          </button>
+        )}
+        {device && copiedSlug === copiedAttribute && <CopyTooltip />}
+      </div>
+    );
+  };
+
+  return (
+    <main className="prime-home device-lookup-container">
+      <div className="grid-container">
+        <div className="grid-row">
+          <div className="prime-container card-container">
+            <div className="usa-card__header">
+              <div>
+                <h2>{props.formTitle}</h2>
+              </div>
+              <div
+                style={{
+                  display: "flex",
+                  justifyContent: "center",
+                  alignItems: "center",
+                }}
+              ></div>
+            </div>
+            <div className="usa-card__body margin-top-1">
+              {props.deviceOptions ? (
+                <div className="grid-row grid-gap">
+                  <div className="tablet:grid-col">
+                    <LabeledText label={"Select device"} />
```

Comment:
This isn't actually a label and needs to be associated with the combobox below. `<label htmlFor={"selectDevice"}>`